src/Form/selectors.js

```diff
@@ -248,7 +273,7 @@ const ID_MATCHERS_LIST = [
         type: 'fullName',
         selector: ID_NAME_SELECTOR,
         matcherFn: (string) =>
-            /\bname\b/i.test(string) && !/company|org/i.test(string)
+            /^(full.?|whole\s)?name\b/i.test(string) && !/company|org/i.test(string)
```

Comment:
The simple "name" here was causing a lot of false positives (project name, document name, you _name_ it). Had to be more specific.